waltz-data/src/main/java/com/khartec/waltz/data/data_flow_decorator/DataFlowDecoratorDao.java

```diff
@@ -76,6 +78,10 @@ public DataFlowDecoratorDao(DSLContext dsl) {
                 .from(DATA_FLOW_DECORATOR)
                 .innerJoin(DATA_FLOW)
                 .on(DATA_FLOW.ID.eq(DATA_FLOW_DECORATOR.DATA_FLOW_ID))
+                .innerJoin(sourceAppAlias)
+                .on(sourceAppAlias.ID.eq(DATA_FLOW.SOURCE_ENTITY_ID))
+                .innerJoin(targetAppAlias)
+                .on(targetAppAlias.ID.eq(DATA_FLOW.TARGET_ENTITY_ID))
```

Comment:
quick comment needed to explain purpose of this join as non-intuitive at first glance